src/components/Sum.jsx

```diff
@@ -0,0 +1,24 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+function Sum({ a, b }) {
+  const sum = a + b;
+
+  return (
+    <p>
+      {`Sum of ${a} and ${b} is ${sum}`}
+    </p>
+  );
+}
```

Comment:
Do you need a sum variable?

Reply:
@kurtkobein It's actually better to move all the calculation and other logic from jsx code to make it more readable. So, yeah, using variable for sum is good here